readinglist/tests/support.py

```diff
@@ -109,6 +109,13 @@ def test_invalid_record_raises_error(self):
                                   status=400)
         self.assertIn('errors', resp.json)
 
+    def test_create_with_invalid_json(self):
+        body = "{'foo': 'bad json'}"
+        self.app.post(self.collection_url,
+                      body,
+                      headers=self.headers,
+                      status=400)
```

Comment:
It will be the case with #24 also there is an helper to test  that `assertFormattedError`